### ISSUE TITLE
Workspace.get is deprecated.

### ISSUE THREAD

**Issue by ooJerryLeeoo** [timomeh/hey-pane#7]
@timomeh After fix #6 I got this deprecation:

`atom.workspace.paneContainer` has always been private, but it is now gone. Please use `atom.workspace.getCenter()` instead and consult the workspace API docs for public methods.
```
Workspace.get (<embedded>:24619:18)
resursiveSet (/Users/jerrylee/.atom/packages/hey-pane/lib/hey-pane.js:105:48)
Object.doFocus (/Users/jerrylee/.atom/packages/hey-pane/lib/hey-pane.js:112:5)
Object.toggleFocus (/Users/jerrylee/.atom/packages/hey-pane/lib/hey-pane.js:70:52)
atom-workspace.heyPaneToggleFocusOfActivePane (/Users/jerrylee/.atom/packages/hey-pane/lib/hey-pane.js:37:22)
CommandRegistry.handleCommandEvent (<embedded>:18796:35)
```

**Comment by timomeh** [comment]
Thank you! Since `getCenter()` isn't available in Atom Stable, I'll fix this deprecation after 1.17 becomes stable.

**Comment by timomeh** [comment]
Closed with 37431bcceda305bd933ff04c3283b2de63b2f9ba
